client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -53,6 +53,27 @@
 	}
 }
 
+.checkout-thank-you__domain-mapping-details {
+	.purchase-detail__description {
+		margin: 0;
+	}
+	.purchase-detail__image {
+		@include breakpoint-deprecated( '>660px' ) {
```

Comment:
Let's try to use the new Gutenberg mixins in new code: https://github.com/Automattic/wp-calypso/blob/HEAD/docs/coding-guidelines/css.md#media-queries
And https://github.com/WordPress/gutenberg/blob/0f1f5e75408705f0ec014f5d2ea3d9fcc8a97817/packages/base-styles/_breakpoints.scss